drafting/conda-template.py

```diff
@@ -0,0 +1,85 @@
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+
+# Reading the qiime2 recipe file
+qiime2_recipe = sys.argv[1]
+with open(qiime2_recipe) as qiime2_recipe:
+    parsed_recipe = yaml.load(qiime2_recipe, Loader=yaml.FullLoader)
+
+# Creating a new blank conda recipe file or 
+# appending to an existing conda recipe file
+filepath = sys.argv[2]
```

Comment:
Can you add a nice error message here if the caller forgets to provide this argument?